lib/yamllint/linter.rb

```diff
@@ -173,6 +173,13 @@ def parse_recurse(psych_parse_data, is_sequence = false)
             hash_end(@last_key.last)
             is_key = false
             @last_key.pop
+          when 'Psych::Nodes::Alias'
+            is_key = false
```

Comment:
hmm i see the gem just completely didn't handle aliases 😂 ok all this seems reasonable 👍

Reply:
@jamestjw ya, which means this gem has never parse our yml file properly, it might still pass if we have duplicated keys